### ISSUE TITLE
`app.setLoginItemSettings` works incorrectly while packaged with the environment variable `--win32metadata.requested-execution-level=requireAdministrator`

### ISSUE THREAD

**Issue by ch1ny** [maxogden/electron-packager#1366]
### Preflight Checklist

* [x] I have read the [[contribution documentation](https://github.com/electron/electron-packager/blob/main/CONTRIBUTING.md)](https://github.com/electron/electron-packager/blob/main/CONTRIBUTING.md) for this project.
* [x] I agree to follow the [[code of conduct](https://github.com/electron/electron/blob/main/CODE_OF_CONDUCT.md)](https://github.com/electron/electron/blob/main/CODE_OF_CONDUCT.md) that this project follows, as appropriate.
* [x] I have searched the issue tracker for a bug that matches the one I want to file, without success.

### Issue Details

* **Electron Packager Version:**
  * 15.5.1
* **Electron Version:**
  * 17.1.0
* **Operating System:**
  * Windows 10 (21H2)

### Behavior

I'm trying to add a feature to my Electron application to make it automatically run after my computer starting up. And this is my source code:

```js
		ipcMain.on('EXCHANGE_OPEN_AT_LOGIN', (evt, openAtLogin) => {
			if (app.isPackaged) {
				app.setLoginItemSettings({
					openAtLogin,
				});
			}
		});
```

If I packaged my application with command like this , my application can be successfully self-booted.

```powershell
electron-packager ./build SduMeeting --platform=win32 --arch=x64 --out ./dist --asar --overwrite
```

However, to implement certain features, I have to package it with command like this to get administrator privileges. 

```powershell
electron-packager ./build SduMeeting --platform=win32 --arch=x64 --out ./dist --asar --overwrite --win32metadata.requested-execution-level=requireAdministrator
```

The application packaged in this way cannot be opened automatically.

### Additional Information

  Excuse me, I'm a college student from a university in China with poor English. I wish my poor presentation wouldn't cause troubles for you.

**Comment by toantt2k** [comment]
Hey man, have you fixed this error yet?

**Comment by ch1ny** [comment]
> Hey man, have you fixed this error yet?

Yes, I have solved this problem by modifying the Windows registry, but I am not sure that my method is appropriate.
You can see my code from the link below:
[https://github.com/ch1ny/SduMeeting/blob/7ed5a51e2eac5716f16f30639455aa3d762b1260/main.js#L363](https://github.com/ch1ny/SduMeeting/blob/7ed5a51e2eac5716f16f30639455aa3d762b1260/main.js#L363)

There may be other, better ways to work it out.

**Comment by s0ckd3** [comment]
up

**Comment by MarshallOfSound** [comment]
Thanks for the report. app.setLoginItemSettings is an Electron runtime API, and its interaction with a requireAdministrator manifest is an Electron and Windows concern rather than something packager controls, so this is out of scope here. Feel free to reopen if you find a packager-specific defect.
